docs/src/languages/php/extensions.md

```diff
@@ -60,110 +60,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| bcmath           |     |     |     | *   | *   | *   | *   | *   | *   |
+| blackfire        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| bz2              |     |     |     | *   | *   | *   | *   | *   | *   |
+| calendar         |     |     |     | *   | *   | *   | *   | *   |     |
+| ctype            |     |     |     | *   | *   | *   | *   | *   |     |
+| curl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| dba              |     |     |     | *   | *   | *   | *   | *   | *   |
+| dom              |     |     |     | *   | *   | *   | *   | *   |     |
+| enchant          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| event            |     |     |     |     | *   | *   | *   | *   |     |
+| exif             |     |     |     | *   | *   | *   | *   | *   |     |
+| ffi              |     |     |     |     |     |     |     | *   | *   |
+| fileinfo         |     |     |     | *   | *   | *   | *   | *   |     |
+| ftp              |     |     |     | *   | *   | *   | *   | *   |     |
+| gd               | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| gearman          | *   | *   | *   |     |     |     |     |     |     |
+| geoip            | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| gettext          |     |     |     | *   | *   | *   | *   | *   |     |
+| gmp              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| http             | *   | *   |     |     |     |     |     | *   |     |
+| iconv            |     |     |     | *   | *   | *   | *   | *   |     |
+| igbinary         |     |     |     | *   | *   | *   | *   | *   | *   |
+| imagick          | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| imap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| interbase        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| intl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| ioncube          |     |     |     | *   | *   | *   |     |     |     |
+| json             |     |     | *   | *   | *   | *   | *   | *   | *   |
+| ldap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mailparse        |     |     |     | *   | *   | *   |     | *   |     |
+| mbstring         |     |     |     | *   | *   | *   | *   | *   | *   |
+| mcrypt           | *   | *   | *   | *   | *   |     |     |     |     |
+| memcache         | *   | *   | *   |     |     |     |     |     |     |
+| memcached        | *   | *   | *   | *   | *   | *   |     | *   |     |
+| mongo            | *   | *   | *   |     |     |     |     |     |     |
+| mongodb          |     |     |     | *   | *   | *   | *   | *   |     |
+| msgpack          |     |     | *   | *   | *   | *   |     | *   |     |
+| mssql            | *   | *   | *   |     |     |     |     |     |     |
+| mysql            | *   | *   | *   |     |     |     |     |     | *   |
```

Comment:
We don't/shouldn't have ext/mysql on PHP 8.  It's long dead, and if it compiles at all I'm in shock.

Reply:
Ah yes I was misguided by the package name thanks